src/components/Borrow/SwitchBorrow/ActionBorrow.tsx

```diff
@@ -112,11 +154,11 @@ export const ActionBorrow: React.FC<IProp> = ({
 
   const changeCustomCRatio = () => {
```

Comment:
suggestion about all methods and calculated values in this PR: maybe think about using useMemo and useCallback hooks to rerender/recalculate only when certain variables are changing